### PR TITLE
[IOTDB-5926] Fix Metric Module Code Smell

### PR DESCRIPTION
**Pull request by SpriCoder**
Fix metric module code smell

### CODE REVIEW

File: iotdb-core/metrics/interface/src/main/java/org/apache/iotdb/metrics/config/MetricConfig.java [hunk 1]
```diff
@@ -53,7 +53,7 @@ public class MetricConfig {
   private Integer prometheusReporterPort = 9091;
 
   /** The iotdb config for iotdb reporter to push metric data. */
-  private final IoTDBReporterConfig iotdbReporterConfig = new IoTDBReporterConfig();
+  private final IotdbReporterConfig iotdbReporterConfig = new IotdbReporterConfig();
```
Comment (OneSizeFitsQuorum) [inline iotdb-core/metrics/interface/src/main/java/org/apache/iotdb/metrics/config/MetricConfig.java:56]: IoTDB is a common term, so maybe we won't have to change the name

**Review by OneSizeFitsQuorum — COMMENTED** [review]
Awesome work! PTAL

**Review by OneSizeFitsQuorum — APPROVED** [review]
LGTM

**Review by BUAAserein — APPROVED** [review]
LGTM

**Pull request merged**